templates/templates.go

```diff
@@ -247,7 +249,10 @@ type Output struct {
 
 // Write writes the Output to the filesystem as a directory, file or snippet.
 func (o *Output) Write() error {
-	path := config.StepAbs(o.Path)
+	// Replace ${STEPPATH} with the base step path.
+	o.Path = strings.Replace(o.Path, "${STEPPATH}", step.BasePath(), -1)
+
+	path := step.Abs(o.Path)
```

Comment:
I thought that you were going to do the replacement in `step.Abs()`, but it's ok for now.

Reply:
I need to store output on the template type.